README.md

```diff
@@ -14,14 +20,55 @@ The QFSAE team designs its boards using Altium. If you do not have Altium Design
+
+### Part Library Setup
+For the majority of projects the only library required is the JLCPCB Altium component library. It can be located in the [libraries](https://github.com/qfsae/pcb/tree/master/libraries) folder of this repository, under **"JLCSMT_LIB"**. 
+
+#### Adding Parts Libraries
+The JLC library is a linked repository, so start by navigating to the [libraries](https://github.com/qfsae/pcb/tree/master/libraries) folder of this repository, clicking through to the JLC library repository and downloading it. Save the library in a safe directory (ie. not Downloads!) We will be adding it to your Altium installation so it is available for use in all projects.
+
+Next, open the components panel if it is not already visible on the right. This can be done by clicking on the **"Panels"** button in the bottom right corner and choosing **"Components"**. This panel is always useful to have open because it is where you search for parts to place. 
+
+Click on 3-lines icon and choose **"File-based Libraries Preferences"**
+
+![library preferences](https://i.imgur.com/SKV5402.png)
+
+In the pop-up window, make sure you are in the **"Installed"** tab, and click on **"Install"**. In the file chooser, make sure to change the visible items from **"Integrated Libraries"** to **"All Installable Libraries"**. Navigate to the downloaded JLC parts library and select all of the library files (not the folder).
+
+![library preferences window](https://i.imgur.com/gVEcPqD.png)
+
+Your final available libraries panel should look similar to the photo below. Be sure to also keep the Altium default libraries and search for parts when making your schematic through the integrated library (using the components panel) and not the schematic libraries themselves. 
 
 ![Altium Panel](https://i.imgur.com/d9237hu.png)
 
-For a broader selection of high quality footprints please follow the installation guide for the [Celestial Altium Library](https://altiumlibrary.com/GetStarted). The component you are looking for will not always appear in the JLC PCB or Celestial libraries. If this is the case, then use Altium's manufacturer part search feature.
+For more information about available part libraries, see the [documentation](https://www.altium.com/documentation/altium-designer/integratedlibrary-dlg-addremovelibrariesformavailable-file-based-libraries-ad). 
+
+#### Other Libraries
+For a broader selection of high quality footprints we use the open source Celestial Library. To install this library, follow the installation guide for the [Celestial Altium Library](https://altiumlibrary.com/GetStarted). 
+
+The component you are looking for will not always appear in the JLC PCB or Celestial libraries. If this is the case, then use Altium's manufacturer part search feature. Try to avoid using online part footprints wherever possible, as it can become difficult to maintain. 
```

Comment:
Question: Should we mention that if you cannot find a footprint anywhere for a specialized part that there is the option of doing a custom footprint and throwing it in a QFSAE library?

Reply:
I'll add a blurb about how we may start an internal library but have not yet